ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm

```diff
@@ -702,37 +708,24 @@ Mobs that mostly focus on dealing RED damage, they are all a bit more frail than
 	if(grab_ready)
 		if(!istype(target, /mob/living/carbon/human))
 			return ..()
-		CounterGrab(target)
+		return CounterGrab(target)
```

Comment:
CounterGrab has no return value so when it's used int his manner AttackingTarget() will return `null`. Not necessarily an issue due to AttackingTarget() not really having a return in the first place, just a heads up.

Reply:
Made OpenFire call the grab instead, but moved CounterGrab above return as requested.